handler/request_test.go

```diff
@@ -4,43 +4,23 @@ import (
 	"net/http"
 	"testing"
 
+	"github.com/ONSdigital/dp-population-types-api/contract"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
 func TestParseRequest(t *testing.T) {
 
-	Convey("Given a valid query param", t, func() {
-		req, err := http.NewRequest("GET", "http://localhost/areas?offset=4&limit=10&q=test", nil)
-		So(err, ShouldBeNil)
-		var ar areaRequest
-		parseRequest(req, &ar)
-		So(ar, ShouldResemble, areaRequest{
-			Limit:    10,
-			Offset:   4,
-			Category: "test",
-		})
-
-		Convey("Given the limit is set to 0 we get dafault", func() {
-			req, err := http.NewRequest("GET", "http://localhost/areas?offset=4&limit=0&q=test", nil)
-			So(err, ShouldBeNil)
-			var ar areaRequest
-			parseRequest(req, &ar)
-			So(ar, ShouldResemble, areaRequest{
-				Limit:    20,
-				Offset:   4,
-				Category: "test",
-			})
-		})
-	})
-
 	Convey("Given a query param that is not in data structure", t, func() {
 		req, err := http.NewRequest("GET", "http://localhost/areas?offset=4&limit=10&X=test", nil)
 		So(err, ShouldBeNil)
-		var ar areaRequest
-		parseRequest(req, &ar)
-		So(ar, ShouldResemble, areaRequest{
-			Limit:    10,
-			Offset:   4,
+		var ar contract.GetAreasRequest
```

Comment:
for consistency I would rename to req

Reply:
The variable was already called `ar`, it's just the struct has moved package. There's already another variable called `req`.